### DISCUSSION TITLE
How to display in x-text two elements?

### DISCUSSION THREAD

**Discussion by bubnenkoff** [alpinejs/alpine#2632] [1. Help] [answered]
I need to display value from x-data like: 10$
```
<div x-data="{price: 10}">
   <span x-text="price"></span>
</div>
```
I know that I can do:
```
  <span x-text="price"></span>$
```  
But I want to make `$` is part of `span` to make it's styling easier.

How I can do it?

https://jsfiddle.net/L7sb0v3w/

**Comment by Tim-Wils** [comment, marked as answer]
Try this: `x-text="price+'$'"`
